package.json

```diff
@@ -21,10 +21,10 @@
   },
   "homepage": "https://github.com/vuejs/eslint-plugin-vue#readme",
   "peerDependencies": {
-    "eslint": "^2.0.0 || ^3.0.0"
+    "eslint": "^2.0.0 || ^3.0.0 || ^4.0.0"
   },
   "dependencies": {
-    "eslint-plugin-html": "^2.0.0",
-    "eslint-plugin-react": "^6.9.0"
+    "eslint-plugin-html": "^2.0.0 || ^3.0.0",
```

Comment:
Thanks for your input @mysticatea, do you have any sort of eta on the 3.0.0 beta as currently this causes breakage within the workflow of anyone upgrading eslint